dashboard/db/migrate/20160404233038_create_level_concept_difficulties.rb

```diff
@@ -0,0 +1,19 @@
+class CreateLevelConceptDifficulties < ActiveRecord::Migration
+  def change
+    create_table :level_concept_difficulties do |t|
+      t.references :level, index: true, foreign_key: true
+      t.timestamps null: false
+
+      t.integer :sequencing, default: 0, null: false
```

Comment:
It isn't clear to me that default: 0 and null: false are the right options.

Allowing null seems appropriate, if the levelbuilder hasn't explicitly set a difficulty for that level and concept? And I'm not sure what it means for a level to have difficulty zero. That it doesn't encompass that concept? Isn't that better represented by null?